crates/sui-bridge-indexer/src/main.rs

```diff
@@ -54,19 +51,24 @@ async fn main() -> Result<()> {
             .expect("Current directory is invalid.")
             .join("config.yaml")
     };
-
     let config = load_config(&config_path).unwrap();
 
-    // start metrics server
     let (_exit_sender, exit_receiver) = oneshot::channel();
-    let metrics = DataIngestionMetrics::new(&Registry::new());
 
     // Init metrics server
-    let metrics_address = SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), 1000);
-    let registry_service = start_prometheus_server(metrics_address);
-    let prometheus_registry = registry_service.default_registry();
-    mysten_metrics::init_metrics(&prometheus_registry);
-    info!("Metrics server started at port {}", 1000);
```

Comment:
reworked this initialization which I am not sure about and would love someone to take a look and comment if there are issues